content/en/docs/components/pipelines/user-guides/core-functions/run-a-pipeline.md

```diff
@@ -21,7 +21,7 @@ To submit a pipeline to the KFP Dashboard:
   alt="Upload pipeline button"
   class="mt-3 mb-3 border border-info rounded">
 
-3. Upload the pipeline IR YAML to "Upload a file", populate the upload pipeline form, and click "Create".
+3. Upload the pipeline IR YAML file or an archived pipeline as a .zip or .tar.gz file, populate the upload pipeline form, and click “Create”.
```

Comment:
@milosjava can you just revert the change in the surrounding quotes in "Create"?

```suggestion
3. Upload the pipeline IR YAML file or an archived pipeline as a .zip or .tar.gz file, populate the upload pipeline form, and click "Create".
```

Reply:
@milosjava pinging you again about this, just in case you didn't see it.